tools/dashboard-plugin-template/README.template.md

```diff
@@ -111,6 +111,10 @@ Building a new plugin is easy. Before you start, ensure that your `.env` and `.e
 
     _TIP_: you can use the `unlink` command to remove the link between dashboard and the plugin.
 
+7.  Update parameters on a dashboard: `{{packageManager}} run update-plugin-params -- <plugin-object-id>`
```

Comment:
Maybe "Update plugin parameters on a dashboard" would be more consistent with the other points?